### PR TITLE
Added and updated packages to support react@16

### PR DESCRIPTION
**Pull request by michelmittelsteiner**
Added and updated packages to support react@16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,15 +44,17 @@
...
-    "react-addons-test-utils": "^15.1.0",
-    "react-dom": "^15.1.0"
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0",
+    "react-addons-test-utils": "^15.5.0",
+    "react-dom": "^16.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0"
+    "react": "^0.14.0 || ^16.0.0-0"
```
Comment (pbomb) [inline package.json]: Can this just be `"react": ">=0.14.0"`? Otherwise, I don't think React v15 would be considered valid.

Comment (michelmittelsteiner) [reply package.json]: You're right. I change this and will revert the second commit. It was a mistake.

**Comment by brentertz** [comment]
This is an open-source package and as such, any updates should be published to npm rather than making it installable via GitHub.

**Comment by michelmittelsteiner** [comment]
No I am not an employee. I just use the package in my project and need the React 16 compatibility.

**Comment by brentertz** [comment]
Cool - just checking. :) Thanks for the PR.  As I am no longer on this project, I have alerted @esoman and am hopeful that he will take it from here.

**Comment by esoman** [comment]
@michelmittelsteiner, thanks for your contribution.  I will be reviewing soon and should be able to get to this today.

**Comment by esoman** [comment]
@michelmittelsteiner, v3.3.0 has been published.

**Pull request merged**